parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -95,10 +95,17 @@ installStandaloneContainerd() {
     # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
     # v1.4.1 is our lowest supported version of containerd
-    
+    CURRENT_KUBERNETES_VERSION=$(kubectl -version)
+    echo "Current version of kubernetes used is: $CURRENT_KUBERNETES_VERSION"
     #if there is no containerd_version input from RP, use hardcoded version
+    echo "(for all cases): KUBERNETES_VERSION IS ${KUBERNETES_VERSION}"
```

Comment:
did you mean CURRENT_KUBERNETES_VERSION instead of KUBERNETES_VERSION?
the latter variable isnt defined anywhere.

Reply:
I did want KUBERNETES_VERSION there to test the value as it is defined in cse_cmd.sh. The build in the PR ([link](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=46517365&view=logs&j=6e436a35-c994-55aa-033b-9a96e1fbff66&t=fa694a85-6f82-5b17-9708-d28cddf5ec77&l=2844)) says that it is an unbound variable, so the variables in cse_cmd.sh do not seem to be accessible here.

Reply:
The CURRENT_KUBERNETES_VERSION variable that I created is currently empty in the build PR. I am trying to find out other ways to access the k8s version.

Reply:
Thanks @anujmaheshwari1 for the suggestion to check in cse_install.sh.